index.html

```diff
@@ -1,11 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <meta charset="UTF-8">
     <link rel="stylesheet" href="styles.css">
     <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-  
+    <h1 class="heading">✨ Self Care Center ✨</h1>
+    <br>
+    <br>
+    <h2>Which type of message?</h2>
+      <form>
+        <div class="box">
+          <div class="radio-container">
+            <div>
```

Comment:
Is there a more semantic element that could be used instead of a <div>? The term "div soup" gets tossed around when you look at an HTML file and all the containers are div elements. A good rule of thumb: if the contents of a div element is more containers, it's OK to be a div. That means it's only there for styling purposes. If the direct children of a div is actual content, then it should probably be a <section> element or something more semantic